src/Maestro/SubscriptionActorService/MergePolicyCheckResult.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace SubscriptionActorService
+{
+    public enum MergePolicyCheckResult
+    {
+        NoPolicies,
```

Comment:
These enums should have explicit numbers assigned to each value because they are serialized. If these get reordered accidentally we want the values to remain consistent.